Add a PyGMT tutorial on 2021-03-16 (#54)
MIGG-NTU/MIG_Docs: source/groupmeeting/workshops.rst

Some workshops hosted in MIG, with PPTs in MIG Google Drive (``groupmeeting.NTU/workshops.MIGG``).

> PyGMT Workshop, Introduction to Markdown, VS Code and Git
> ---------------------------------------------------------

Reviewer: ```suggestion
PyGMT Workshop
--------------
```

> An introduction to PyGMT:

Reviewer: ```suggestion
`An introduction to PyGMT <https://github.com/MIGG-NTU/PyGMT2021>`__:
```

> - Github repository: https://github.com/MIGG-NTU/PyGMT2021 which contains the installation dependencies and the tutorial file (Jupyter Lab / Jupyter Notebook).
> - Installation instrutions to PyGMT: https://www.pygmt.org/latest/install.html
> - Tutorials regarding basic plots and notes regarding the command-line in GMT compared to PyGMT are discussed.

Reviewer: ```suggestion
- Basic plots
- Difference between the command-line in GMT and PyGMT
- How to use non-wrapped GMT commands
- Some reference tutorial
```

> 

Reviewer: ```suggestion

`Introduction to some useful tools <https://github.com/MIGG-NTU/PyGMT2021/issues/2>`__

- Markdown
- VS Code
- Git
```
